deployment/bare_metal/nginx/sites-available/gradido-federation.conf.template

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
it fulfills inserting a linefeed to ensure that the content for each api-version loop inning is starting in a new line, because of attaching ">>"  the content in the same file.

Reply:
What about putting it at the end of file?

Reply:
no, at the beginning it will ensure that the first apiversion replacement starts with a new line